### ISSUE TITLE
Commit Graph content not displayed

### ISSUE THREAD

**Issue by kmbgit** [gitkraken/vscode-gitlens#2179]
### Description

Using Remote:SSH extension v0.84.0 to connect to a CentOS 7 linux server running git 2.18.2 for RedHat from Software Collections.
GitLens v12.2.0 with Pro subscription.

Git repo is a clone from from a local GitLab Enterprise Edition 13.12.0-ee instance.

When I click on the "Visualize commits on the all-now Commit Graph" icon on the status bar I get a new Commit Graph tab but there's no content.  When I click on the Refresh icon in that tab I see a popup error message "Cannot write private member to an object whose class did not declare it".

### GitLens Version

12.2.0

### VS Code Version

Version: 1.70.2 (user setup)
Commit: e4503b30fc78200f846c62cf8091b76ff5547662
Date: 2022-08-16T05:35:13.448Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.19044

### Git Version

git version 2.18.4 for RedHat by Software Collections

### Logs, Screenshots, Screen Captures, etc

[2022-08-31 10:49:44.620] [GIT  ] [/home/kevin/GitLab/satixfy-hub/nms-emulator] git merge-base --fork-point  master • Command failed: /opt/rh/rh-git218/root/usr/bin/git -c core.quotepath=false -c color.ui=false merge-base --fork-point  master • No such ref: '' • 9 ms

[2022-08-31 10:49:44.621] /home/kevin/.vscode-server/extensions/eamodio.gitlens-12.2.0/dist/gitlens.js:12:8249
Error: Command failed: /opt/rh/rh-git218/root/usr/bin/git -c core.quotepath=false -c color.ui=false merge-base --fork-point  master
fatal: No such ref: ''

[2022-08-31 10:49:44.629] [GIT  ] [/home/kevin/GitLab/satixfy-hub/nms-emulator] git merge-base  master • Command failed: /opt/rh/rh-git218/root/usr/bin/git -c core.quotepath=false -c color.ui=false merge-base  master • Not a valid object name • 8 ms

[2022-08-31 10:49:44.630] /home/kevin/.vscode-server/extensions/eamodio.gitlens-12.2.0/dist/gitlens.js:12:8249
Error: Command failed: /opt/rh/rh-git218/root/usr/bin/git -c core.quotepath=false -c color.ui=false merge-base  master
fatal: Not a valid object name 

[2022-08-31 13:16:24.472] [  1d3] SettingsWebview(gitlens.settings).postMessage — i=(id=host:1, method=configuration/didChange)
[2022-08-31 13:16:24.473] [  1d4] WelcomeWebview(gitlens.welcome).postMessage — i=(id=host:2, method=configuration/didChange)
[2022-08-31 13:16:24.559] [  1d3] SettingsWebview(gitlens.settings).postMessage completed • 87 ms
[2022-08-31 13:16:24.559] [  1d4] WelcomeWebview(gitlens.welcome).postMessage completed • 86 ms
[2022-08-31 13:16:29.138] [  1d5] GraphWebview(gitlens.graph).refresh — i=true
[2022-08-31 13:16:29.166] [  1d6] GitProviderService.getCommitsForGraph — i=/home/kevin/GitLab/satixfy-hub/nms-emulator, e=asWebviewUri(l){return Ze(this,w,!0,"f"),(0,C.asWebviewUri)(l,Me(this,S,"f"))}, t={"limit":500,"ref":"HEAD"}
[2022-08-31 13:16:29.166] [  1d7] LocalGitProvider.getCommitsForGraph — e=/home/kevin/GitLab/satixfy-hub/nms-emulator, t=asWebviewUri(l){return Ze(this,w,!0,"f"),(0,C.asWebviewUri)(l,Me(this,S,"f"))}, r={"limit":500,"ref":"HEAD"}
[2022-08-31 13:16:29.166] [  1d8] LocalGitProvider.getStash — e=/home/kevin/GitLab/satixfy-hub/nms-emulator
[2022-08-31 13:16:29.166] [  1d8] LocalGitProvider.getStash completed • 0 ms
[2022-08-31 13:16:29.166] [  1d9] LocalGitProvider.getRemotes — e=/home/kevin/GitLab/satixfy-hub/nms-emulator
[2022-08-31 13:16:29.187] [GIT  ] [/home/kevin/GitLab/satixfy-hub/nms-emulator] git log -z --format=%H%x00%ct%x00 --full-history -M -m -n2 --first-parent HEAD -- • 20 ms
[2022-08-31 13:16:29.188] [  1da] LocalGitProvider.getLog — e=/home/kevin/GitLab/satixfy-hub/nms-emulator, t={"all":true,"ordering":"date","limit":500,"stdin":""}
[2022-08-31 13:16:29.196] [GIT  ] [/home/kevin/GitLab/satixfy-hub/nms-emulator] git remote -v • 29 ms
[2022-08-31 13:16:29.196] [  1db] GitRemoteParser.parse completed • 0 ms
[2022-08-31 13:16:29.196] [  1d9] LocalGitProvider.getRemotes completed • 29 ms
[2022-08-31 13:16:29.216] [GIT  ] [/home/kevin/GitLab/satixfy-hub/nms-emulator] git log --format=%x3c%x2ff%x3e%n%x3cr%x3e%x20%H%n%x3ca%x3e%x20%aN%n%x3ce%x3e%x20%aE%n%x3cd%x3e%x20%at%n%x3cn%x3e%x20%cN%n%x3cm%x3e%x20%cE%n%x3cc%x3e%x20%ct%n%x3cp%x3e%x20%P%n%x3ct%x3e%x20%D%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e --name-status --full-history -M -m --all --date-order -n501 -- • 27 ms
[2022-08-31 13:16:29.216] [  1dc] LocalGitProvider.getCurrentUser — e=/home/kevin/GitLab/satixfy-hub/nms-emulator
[2022-08-31 13:16:29.217] [  1dc] LocalGitProvider.getCurrentUser completed • 0 ms
[2022-08-31 13:16:29.217] [  1dd] GitLogParser.parse
[2022-08-31 13:16:29.222] [  1dd] GitLogParser.parse completed • 4 ms
[2022-08-31 13:16:29.222] [  1da] LocalGitProvider.getLog completed • 33 ms
[2022-08-31 13:16:29.222] [  1d7] LocalGitProvider.getCommitsForGraph — failed • 56 ms
TypeError: Cannot write private member to an object whose class did not declare it
[2022-08-31 13:16:29.222] [  1d6] GitProviderService.getCommitsForGraph — failed • 56 ms
TypeError: Cannot write private member to an object whose class did not declare it
[2022-08-31 13:16:29.223] [  1d5] GraphWebview(gitlens.graph).refresh — failed • 84 ms
TypeError: Cannot write private member to an object whose class did not declare it
[2022-08-31 13:16:40.585] [  1de] GitProviderService.onWindowStateChanged — i=focused=false
[2022-08-31 13:16:40.585] [  1de] GitProviderService.onWindowStateChanged completed • 0 ms
[2022-08-31 13:16:43.937] [  1df] GitProviderService.onWindowStateChanged — i=focused=true
[2022-08-31 13:16:43.937] [  1df] GitProviderService.onWindowStateChanged completed • 0 ms
[2022-08-31 13:16:43.937] [  1e0] SubscriptionService.validate
[2022-08-31 13:16:43.937] [  1e1] SubscriptionService.ensureSession — i=false
[2022-08-31 13:16:43.938] [  1e1] SubscriptionService.ensureSession completed • 0 ms
[2022-08-31 13:16:43.938] [  1e2] SubscriptionService.checkInAndValidateCore — i=kmbgit
[2022-08-31 13:16:44.370] [  1e3] SubscriptionService.validateSubscription — i={"user":{"id":"8c2ca5d7-708d-4702-a511-ae2ddd257d5d","email":"kevin@broadey.net","status":"activated","firstGitLensCheckIn":"2022-08-31T10:46:16.854Z"},"licenses":{"paidLicenses":{"gitlens-pro":{"latestStatus":"active","latestStartDate":"2022-08-31T07:00:00.000Z","latestEndDate":"2023-08-31T06:59:59.000Z"}},"effectiveLicenses":{}}}
[2022-08-31 13:16:44.370] [  1e4] SubscriptionService.changeSubscription — i={"plan":{"actual":{"id":"pro","name":"GitLens+ Pro","startedOn":"2022-08-31T07:00:00.000Z","expiresOn":"2023-08-31T06:59:59.000Z"},"effective":{"id":"pro","name":"GitLens+ Pro","startedOn":"2022-08-31T07:00:00.000Z","expiresOn":"2023-08-31T06:59:59.000Z"}},"state":5,"previewTrial":{"startedOn":"2022-08-31T10:19:27.459Z","expiresOn":"2022-09-03T22:59:59.999Z"},"account":{"id":"8c2ca5d7-708d-4702-a511-ae2ddd257d5d","email":"kevin@broadey.net","verified":true}}
[2022-08-31 13:16:44.370] [  1e4] SubscriptionService.changeSubscription completed • 0 ms
[2022-08-31 13:16:44.370] [  1e3] SubscriptionService.validateSubscription completed • 0 ms
[2022-08-31 13:16:44.370] [  1e2] SubscriptionService.checkInAndValidateCore completed • 432 ms
[2022-08-31 13:16:44.370] [  1e0] SubscriptionService.validate completed • 433 ms
[2022-08-31 13:16:51.399] [  1e5] TimelineWebviewView(gitlens.views.timeline).onActiveEditorChanged
[2022-08-31 13:16:51.399] [  1e5] TimelineWebviewView(gitlens.views.timeline).onActiveEditorChanged completed • 0 ms
[2022-08-31 13:17:10.005] [  1e6] TimelineWebviewView(gitlens.views.timeline).onActiveEditorChanged
[2022-08-31 13:17:10.005] [  1e6] TimelineWebviewView(gitlens.views.timeline).onActiveEditorChanged completed • 0 ms
[2022-08-31 13:17:14.978] [  1e7] TimelineWebviewView(gitlens.views.timeline).onActiveEditorChanged
[2022-08-31 13:17:14.978] [  1e7] TimelineWebviewView(gitlens.views.timeline).onActiveEditorChanged completed • 0 ms

**Comment by TCROC** [comment]
I also have this issue on our Azure DevOps repo.  We also have submodules so idk if that has any impact

**Comment by kitcatdotone** [comment]
Have the same issue.

Stack trace:
```
[2022-09-01 14:10:40.852] [exthost] [error] TypeError: Cannot write private member to an object whose class did not declare it
	at Ze (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:97:6683)
	at asWebviewUri (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:87:185125)
	at getRemoteIconUri (/Users/me/.vscode/extensions/eamodio.gitlens-12.2.0/dist/gitlens.js:197:275041)
	at He.getCommitsForGraphCore (/Users/me/.vscode/extensions/eamodio.gitlens-12.2.0/dist/gitlens.js:20:2062)
	at He.getCommitsForGraph (/Users/me/.vscode/extensions/eamodio.gitlens-12.2.0/dist/gitlens.js:20:1197)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)
	at async Fn.getState (/Users/me/.vscode/extensions/eamodio.gitlens-12.2.0/dist/gitlens.js:185:16499) gitlens.refreshGraphPage {"value":"eamodio.gitlens","_lower":"eamodio.gitlens"}
```

Something wrong with this block in `.vscode/extensions/eamodio.gitlens-12.2.0/dist/gitlens.js`:

```js
function f(U, A, k, g = l.window.activeColorTheme) {
        var x;
        if (((x = A.provider) == null ? void 0 : x.icon) == null) return;
        const $ = l.Uri.joinPath(
          U.context.extensionUri,
          `images/${(0, u.GC)(g) ? "light" : "dark"}/icon-${
            A.provider.icon
          }.svg`
        );
        return k != null ? k($) : $;
}
a(f, "getRemoteIconUri");
```

**Comment by pauljohnleonard** [comment]
Same here

**Comment by IBonkI** [comment]
Same

**Comment by eamodio** [comment]
Should be fixed in 12.2.1 that just went live.
